builds/linux/rhel/9/linux-rhel.pkrvars.hcl.example

```diff
@@ -21,3 +21,20 @@ vm_firmware              = "efi-secure"
 iso_datastore_path       = "iso/linux/rhel"
 iso_content_library_item = "rhel-9.4-x86_64-dvd"
 iso_file                 = "rhel-9.4-x86_64-dvd.iso"
+
+rpm_gpg_keys = [
```

Comment:
Should this only be used if you want to use specific GPG keys or does this need to be new requirement?

Reply:
by default, it seems there is no gpg key in RedHat